client/state/partner-portal/payment-methods/actions.ts

```diff
@@ -1,9 +1,5 @@
 import { AnyAction } from 'redux';
 
-export function changeBrand( payload: string ): AnyAction {
-	return { type: 'BRAND_SET', payload };
-}
-
```

Comment:
Out of curiosity, why are we deleting this?

Reply:
The brand logo (e.g., Visa, Mastercard, etc.) is handled manually in Calypso, but utilizing the shorthand Stripe pattern for the field, we no longer need this logic.